test/e2e/testdata/acme/deployment.yaml

```diff
@@ -26,11 +26,11 @@ spec:
               protocol: TCP
             - containerPort: 8888
               protocol: TCP
+          securityContext:
```

Comment:
What is the issue faced in having security context at pod level rather than moving to container specific?

Reply:
Yeah, properties `allowPrivilegeEscalation`, `capabilities` are not part of pod securityContext, but container, so moved to it. We can add other properties defined for pod as well, but since just one container is defined, thought to keep just the current changes. Please let me know WDYT.